CHANGELOG.txt

```diff
@@ -2,6 +2,12 @@
 TextWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+Feb 23, 2022 1.3.1
+-------------------------------------------------------------------------------
+minor:
+- Updated dependency requirements for flair to pass python3.9 unit tests
```

Comment:
minor suggestion, can we be explicit here and mention what the update is? Otherwise, someone has to go back in commits to figure out what's needed for 3.9. Just add sth like the upper bound has to be xxx for flair to pass, sth like that :)